python_legacy/tox.ini

```diff
@@ -49,7 +49,7 @@ commands =
 basepython = python3
 skip_install = true
 deps =
-    flake8>=3.8.4
+    flake8==5.0.4
     flake8-import-order>=0.9
```

Comment:
should we pin this as well maybe, since in the future it might get upgraded to a version requiring flake8 6.x ?
would follow the style of all other deps here

Reply:
Makes sense 👍🏻 
```suggestion
    flake8-import-order==0.18.1
```